src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/integration_tests/tests/IntegrationTest.kt

```diff
@@ -1,12 +1,16 @@
 package org.vaccineimpact.orderlyweb.tests.integration_tests.tests
 
+import com.nhaarman.mockito_kotlin.doReturn
+import com.nhaarman.mockito_kotlin.mock
 import org.assertj.core.api.Assertions
 import org.junit.After
 import org.junit.Before
 import org.junit.BeforeClass
```

Comment:
unused imports

Reply:
Sorry, which ones are unused? Nothing in this file getting flagged as unused in my IDE. `After`, `Before` and `BeforeClass` are all used e.g. [here](https://github.com/vimc/orderly-web/pull/393/files#diff-f6d2ea89bba90ff1d923392013f699127f919711d17db243a1e6bd18ad8d21ceR47)

Reply:
Sorry that link might not work - line 47 has `@Before`